content/en/blog/2025/go-auto-instrumentation-beta.md

```diff
@@ -0,0 +1,127 @@
+- **Kafka-go instrumentation**: Monitor and trace Kafka messaging using the
+  [`kafka-go` package](https://pkg.go.dev/github.com/segmentio/kafka-go).
+- **Extensible with OpenTelemetry’s Trace API**: Seamlessly extend
+  auto-instrumentation with custom spans using the
+  [OpenTelemetry Go Trace API](https://pkg.go.dev/go.opentelemetry.io/otel).
+- **Configuration using environment variables**: Simplify configuration with
+  environment-based settings, reducing the need for code changes.
+- **Semantic convention compliance**: Produced telemetry complies with the
+  latest OpenTelemetry semantic conventions ensuring compatibility with the OTel
+  ecosystem.
+
+## Getting started
+
+Getting started with OpenTelemetry Go Auto-Instrumentation is straightforward!
+For detailed instructions on installation, configuration, and running your
+application with auto-instrumentation, check out the
+[Getting Started guide](https://github.com/open-telemetry/opentelemetry-go-instrumentation/blob/main/docs/getting-started.md).
+
+To see a complete example, check out the
+[`rolldice` application](https://github.com/open-telemetry/opentelemetry-go-instrumentation/tree/0ebb7f21116bfdd8e29c315efdbf359cd74bddac/examples/rolldice).
+
+## The road to stabilization
+
+With the beta release now available, the project team is focused on preparing
+for a stable release. Over the coming year, we’ll concentrate on achieving the
+following goals:
+
+1. **Optimize runtime instrumentation with eBPF**
+
+   We will continue to leverage and improve eBPF for dynamic runtime
+   instrumentation, ensuring that Go applications have reliable, low-overhead
+   observability. This includes staying up to date with the latest developments
+   in both the Go and eBPF ecosystems.
+
+2. **Expand ecosystem support**
+
+   While the beta release supports a limited number of Go packages and only a
+   single telemetry processing pipeline, we plan to broaden this support. We’ll
+   introduce instrumentation for additional Go packages and allow users to
+   create custom telemetry processing pipelines. This expansion will make it
+   easier to integrate with popular Go packages and provide flexibility for
+   custom instrumentation.
+
+3. **Integrate with Beyla’s donation to OpenTelemetry**
+
+   The
+   [donation of Beyla to OpenTelemetry](https://github.com/open-telemetry/community/issues/2406)
+   brings the opportunity to unify and enhance OpenTelemetry's eBPF
+   auto-instrumentation offerings. As an eBPF-based tool for both traces and
+   metrics, this donation will help provide instrumentation for more signals,
+   protocols, and languages in open source. And because the Go instrumentation
+   in Beyla is already based on the OpenTelemetry Go Auto-Instrumentation
+   libraries, we plan to merge the projects – and their development teams –
+   under the OpenTelemetry community organization.
+
+   The integration of Beyla will significantly enhance auto-instrumentation for
+   Go!
```

Comment:
@open-telemetry/governance-committee @open-telemetry/technical-committee PTAL, if I understand it correctly the donation is not yet accepted, so I am not sure if this block needs to be rephrased, or if we need to wait for a little bit how it evolves?

Reply:
I'm happy to rephrase. Do we just want to say that the OTel Go auto-instrumentation SIG plans to support this donation when it happens? Or did you have another suggestion?

Reply:
Maybe "The OTel Go auto-instrumentation SIG and the Beyla project owners are collaborating on a potential donation, which can significantly enhance auto-instrumentation for Go".

Reply:
I've rephrased. PTAL.

Reply:
thanks!